interfaces/Portal/src/app/components/program-card/program-card.component.html

```diff
@@ -32,35 +32,25 @@ <h1>
   [routerLink]="['/program', program.id, program.phase]"
 >
   <ion-card-header>
-    <ion-row class="ion-padding-horizontal">
-      <ion-col size="10">
-        <ion-card-title color="primary">
-          <strong class="program-title">{{ program.titlePortal }}</strong>
-        </ion-card-title>
-        <ion-card-subtitle color="medium">
-          <small>
-            <strong>
-              {{ 'page.programs-list.card.updated' | translate }}
-              {{ program.updated | date: DateFormat.dayAndTime }}
-            </strong>
-          </small>
-        </ion-card-subtitle>
-      </ion-col>
-      <ion-col
-        size="2"
-        class="ion-no-padding ion-text-end"
-      >
-        <!-- <ion-button fill="clear" color="dark" size="small">
-          <ion-icon name="ellipsis-horizontal"></ion-icon>
-        </ion-button> -->
-      </ion-col>
-    </ion-row>
+    <ion-card-title color="primary">
+      <strong class="program-title">{{ program.titlePortal }}</strong>
+    </ion-card-title>
+    <ion-card-subtitle color="medium">
+      <small>
+        <strong>
+          {{ 'page.programs-list.card.updated' | translate }}
+          {{ program.updated | date: DateFormat.dayAndTime : locale }}
+        </strong>
```

Comment:
Could maybe be combined in 1 translatable-string with a "{{date}}"-placeholder.

```suggestion
        <strong>
          {{
            'page.program-list.card.updated' |
              translate: {
                date: (program.updated | date: DateFormat.dayAndTime : locale)
              }
          }}  
        </strong>
```

That way the position of the date + its surrounding syntax (":") can be adjusted per language.